Autonomous decision making demonstration.ipynb

```diff
@@ -153,8 +153,8 @@
     }
    ],
    "source": [
-    "for i in range(len(obstacles)):\n",
-    "    arr = applyGaussianToField(arr,obstacles[i][3],obstacles[i][0],obstacles[i][1],obstacles[i][2])\n",
+    "for obstacle in obstacles:\n",
+    "    arr = applyGaussianToField(arr,*obstacle)\n",
```

Comment:
Wont this mess up the order in which things have been passed in? Are you sure this works?

Reply:
I changed the order from 3,0,1,2 to 0,1,2,3 to facilitate the * shorthand

Reply:
I cant see that anywhere in the changes?